xero-app-store.yaml

```diff
@@ -299,3 +397,93 @@ components:
         usageUnit:
           description: The unit of the usage product. e.g. "user", "minutes", "SMS", etc
           type: string
+    UsageRecord:
+      properties:
+        quantity:
+          description: The quantity recorded
+          format: int32
+          type: integer
+        subscriptionId:
+          description: The unique identifier of the Subscription.
+          format: guid
+          type: string
+        subscriptionItemId:
+          description: The unique identifier of the SubscriptionItem.
+          format: guid
+          type: string
+        testMode:
+          description: If the subscription is a test subscription
+          type: boolean
+        timeStamp:
+          description: The timestamp when this usage occurred
+          format: date-time
+          type: string
+        usageRecordId:
+          description: The unique identifier of the usageRecord.
+          format: guid
+          type: string
+      required:
+        - usageRecordId
+        - subscriptionId
+        - subscriptionItemId
+        - quantity
+        - testMode
+        - timeStamp
+      type: object
+    UsageRecordsListItem:
+      properties:
+        pricePerUnit:
+          description: The price per unit
+          format: decimal
+          type: number
+        productId:
+          description: The unique identifier of the linked Product
+          format: guid
+          type: string
+        quantity:
+          description: The quantity recorded
+          format: int32
```

Comment:
How is this quantity different from the quantity in usageRecord?

Reply:
It's not. they're the same

Reply:
What should we put in the description to make this clear?

Reply:
I'm not sure how to address your concern because there's nothing to do if they're the same.
Please provide an example to more clearly convey your point.

Reply:
Let's review this again later through a Jira ticket that you'll be creating as requested here: https://xero.slack.com/archives/C010VN4E3HT/p1666306720248679?thread_ts=1666141336.405279&cid=C010VN4E3HT

Reply:
The object "UsageRecord" is not native to this request. It does not belong in this section of the documentation.

Reply:
Both `UsageRecord` and `UsageRecordsListItem` are for capturing the payload for the single usage record. It does not make sense to mention that the `Quantity` is used in one or other responses in the description.